### PR TITLE
adding new upn param

### PR DESCRIPTION
**Pull request by maimorag**
## Contributing to Cortex XSOAR Content
Make sure to register your contribution by filling the [contribution registration form](https://forms.gle/XDfxU4E61ZwEESSMA)

**The Pull Request will be reviewed only after the contribution registration form is filled.**

## Status
- [x] In Progress

## Related Issues
fixes: https://jira-hq.paloaltonetworks.local/browse/CIAC-4228

## Description
A few sentences describing the overall goals of the pull request's commits.

## Screenshots
Paste here any images that will help the reviewer

## Minimum version of Cortex XSOAR
- [X] 5.0.0

## Must have
- [ ] Tests
- [ ] Documentation

### CODE REVIEW

File: Packs/EWS/Integrations/EWSO365/EWSO365.yml [hunk 1]
```diff
@@ -24,25 +24,30 @@ configuration:
   name: default_target_mailbox
   required: true
   type: 0
-- additionalinfo: Supports Exchange Folder ID and sub-folders e.g. Inbox/Phishing.
-  defaultvalue: Inbox
+- additionalinfo: UPN mailbox to run commands on and to fetch incidents from.
```
Comment (Shellyber) [inline Packs/EWS/Integrations/EWSO365/EWSO365.yml:27]:
```suggestion
- additionalinfo: If this parameter is given, the commands will run with the UPN mailbox instead of the default target mailbox.
```

File: Packs/EWS/Integrations/EWSO365/README.md [hunk 2]
```diff
@@ -2,6 +2,8 @@ Exchange Web Services (EWS) provides the functionality to enable client applicat
 
 The EWS O365 integration implants EWS leading services. The integration allows getting information on emails and activities in a target mailbox, and some active operations on the mailbox such as deleting emails and attachments or moving emails from folder to folder.
 
+The integration will use the UPN parameter as the target mailbox if it's different from the Email Address, otherwise, the Email Address is used.
```
Comment (Shellyber) [inline Packs/EWS/Integrations/EWSO365/README.md:5]:
```suggestion
The integration will use the UPN parameter (if given) as the target mailbox if it's different from the Email Address, otherwise, the Email Address is used.
```

File: Packs/EWS/ReleaseNotes/2_0_5.md [hunk 3]
```diff
@@ -2,4 +2,4 @@
 #### Integrations
 ##### EWS O365
 - Added the **Mark fetched emails as read** parameter to the integration instance configuration.
```
Comment (Shellyber) [inline Packs/EWS/ReleaseNotes/2_0_5.md:4]: Why this file is here? revert the changes

File: Packs/EWS/ReleaseNotes/2_0_6.md [hunk 4]
```diff
@@ -0,0 +1,4 @@
+
+#### Integrations
+##### EWS O365
+- Added the parameter **UPN** to configuration, in order to fix fetch.
```
Comment (Shellyber) [inline Packs/EWS/ReleaseNotes/2_0_6.md:4]:
```suggestion
- Added the *UPN* parameter to the integration instance configuration.
```

File: Packs/EWS/ReleaseNotes/2_0_5.md (hunk 3 above)
Comment (maimorag) [reply Packs/EWS/ReleaseNotes/2_0_5.md:4]: from master (not my change), I added extra space by mistake

**Review by Shellyber — COMMENTED** [review]
Looking good.

File: Packs/EWS/Integrations/EWSO365/EWSO365.py [hunk 5]
```diff
@@ -2384,18 +2383,17 @@ def sub_main():
...
+    if params.get('upn_mailbox') and not(args.get('target_mailbox')):
+        params['default_target_mailbox'] = params.get('upn_mailbox')
     try:
         client = EWSClient(**params)
         start_logging()
-
         # replace sensitive access_token value in logs
         add_sensitive_log_strs(client.credentials.access_token.get('access_token', ''))
 
-        command = demisto.command()
```
Comment (Shellyber) [inline Packs/EWS/Integrations/EWSO365/EWSO365.py:2398]: why did you remove this?

Comment (maimorag) [reply Packs/EWS/Integrations/EWSO365/EWSO365.py:2398]: I moved it next to the args but since I don't need it I'll return it back

**Review by Shellyber — APPROVED** [review]
